### PR TITLE
Mccalluc/faster reading

### PR DESCRIPTION
**Pull request by mccalluc**
- Parsing of gz files about 3 times as fast
- Handle zipped gct files
- Expanded docs
- Plain zip files are unsupported until we have a clear usecase

### CODE REVIEW

File: dataframer/dataframer.py [hunk 1]
```diff
@@ -15,51 +47,38 @@ def parse(file, col_zero_index=True, keep_strings=False, relabel=False):
...
-                file,
-                index_col=index_col,
-                dialect=dialect
-            )
-        if is_gct:
-            dataframe.drop(columns=['Description'], inplace=True)
-            # TODO: Combine the first two columns?
+    if sniff_result.is_gct:
+        dataframe.drop(columns=['Description'], inplace=True)
+        # TODO: Combine the first two columns?
```
Comment (jkmarx) [inline dataframer/dataframer.py]: fix todo, bring up in dev meeting

File: tests/test_dataframer.py [hunk 2]
```diff
@@ -54,10 +54,11 @@ def test_read_gzip(self):
         self.assert_file_read(
             b'\x1f\x8b\x08\x08\xe5\xf2\x82Z\x00\x03fake.csv\x00\xd3I\xd2I\xe62\xd41\xd21\x06\x00\xfb\x9a\xc9\xa6\n\x00\x00\x00', self.target)  # noqa: E501
 
-    def test_read_zip(self):
-        self.assert_file_read(
-            b'PK\x03\x04\n\x00\x00\x00\x00\x00\x8dZML\xfb\x9a\xc9\xa6\n\x00\x00\x00\n\x00\x00\x00\x08\x00\x1c\x00fake.csvUT\t\x00\x03J\x10\x83Zk\x11\x83Zux\x0b\x00\x01\x04\xf6\x01\x00\x00\x04\x14\x00\x00\x00,b,c\n1,2,3PK\x01\x02\x1e\x03\n\x00\x00\x00\x00\x00\x8dZML\xfb\x9a\xc9\xa6\n\x00\x00\x00\n\x00\x00\x00\x08\x00\x18\x00\x00\x00\x00\x00\x01\x00\x00\x00\xa4\x81\x00\x00\x00\x00fake.csvUT\x05\x00\x03J\x10\x83Zux\x0b\x00\x01\x04\xf6\x01\x00\x00\x04\x14\x00\x00\x00PK\x05\x06\x00\x00\x00\x00\x01\x00\x01\x00N\x00\x00\x00L\x00\x00\x00\x00\x00', self.target  # noqa: E501
-        )
+    # No use-case for zip files right now, but it could be brought back.
```
Comment (jkmarx) [inline tests/test_dataframer.py]: personal preference to remove commented code

**Review by jkmarx — APPROVED** [review]
Plus update text we discuss in chat

File: dataframer/dataframer.py (hunk 1 above)
Comment (mccalluc) [reply dataframer/dataframer.py]: I'll just drop the TODO: My sense is that ignoring `Description` makes sense for now, and if that changes, we'll hear about it.

**Pull request merged**